octavia-cli/octavia_cli/api_http_headers.py

```diff
@@ -23,9 +23,11 @@ class ApiHttpHeader:
     value: str
 
     def __post_init__(self):
+        self.name = str(self.name)
+        self.value = str(self.value)
         try:
-            assert isinstance(self.name, str) and self.name
```

Comment:
how come the `isinstance` check was removedf?

Reply:
because above we forced everything to be a string.... so it's a string :p

Reply:
checks out